artifactory/1.11/README.md

```diff
@@ -0,0 +1,26 @@
+# How to use Artifactory on DC/OS
+
+Currently you are only able to use licensed versions of Artifactory on DC/OS. To
+get a trial license, go to [JFrog's
+website](https://www.jfrog.com/artifactory/free-trial-mesosphere/).
+
+Artifactory requires a database named `artifactory` made available (MySQL, Oracle, MS
+SQL Server or Postgres). To set up a testing instance of Postgres on DC/OS, see
+[this guide](install-postgres.md).
```

Comment:
You have MySQL instructions as well, should they be linked here?

Reply:
Probably, Just the Postgres link will be fine.